_i18n/it/get-started/contributing.md

```diff
@@ -10,18 +9,18 @@ Monero is an open-source, community-driven project. Described below are several
                <div class="full col-lg-12 col-md-12 col-sm-12 col-xs-12">
                         <div class="info-block text-adapt">
                             <div class="row center-xs">
-                                <div class="col"><h2>Support the Network</h2></div>
+                                <div class="col"><h2>Supporta la rete</h2></div>
                             </div>
 <div class="row start-xs" markdown="1">
 
-### Develop
-Monero is primarily written in C++. As it is a decentralized project, anyone is welcome to add or make changes to existing code. Pull requests are merged based on community consensus. See the <a href="https://github.com/monero-project"> repositories</a> and outstanding <a href="https://github.com/monero-project/monero/issues">issues</a>.
+### Sviluppa
+Monero è scritto principalmente in C++. Essendo un progetto decentralizzato, chiunque è il/la benvenuto/a e può fare aggiunte o modifiche al codice esistente. Le "Pull request" vengono accettate secondo il consenso della comunità. Vedi i <a href="https://github.com/monero-project">repositori</a> e i <a href="https://github.com/monero-project/monero/issues">problemi</a> in sospeso.
```

Comment:
"repositori" => repositories , we can keep the english word for that since it's a technical word.

Reply:
Agreed.